Examples/Scripts/Python/geometry.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+from common import getOpenDataDetector
+import os
+
+from acts.examples import (
+    GenericDetector,
+    AlignedDetector,
+    WhiteBoard,
+    AlgorithmContext,
+    ProcessCode,
+    CsvTrackingGeometryWriter,
+    ObjTrackingGeometryWriter,
+    JsonSurfacesWriter,
+    JsonMaterialWriter,
+    JsonFormat,
+)
+
+import acts
+
+from acts import MaterialMapJsonConverter
+
+
+def runGeometry(
+    trackingGeometry,
+    decorators,
+    outputDir,
+    events=1,
+    outputObj=True,
+    outputCsv=True,
+    outputJson=True,
+    outputRoot=True,
+):
+
+    for ievt in range(events):
+        eventStore = WhiteBoard(name=f"EventStore#{ievt}", level=acts.logging.INFO)
+        ialg = 0
+
+        context = AlgorithmContext(ialg, ievt, eventStore)
+
+        for cdr in decorators:
+            r = cdr.decorate(context)
+            if r != ProcessCode.SUCCESS:
+                raise RuntimeError("Failed to decorate event context")
+
+        if outputCsv:
+            writer = CsvTrackingGeometryWriter(
+                level=acts.logging.INFO,
+                trackingGeometry=trackingGeometry,
+                outputDir=os.path.join(outputDir, "csv"),
+                writePerEvent=True,
+            )
+            writer.write(context)
+
+        if outputObj:
+            writer = ObjTrackingGeometryWriter(
+                level=acts.logging.INFO, outputDir=os.path.join(outputDir, "obj")
+            )
+            writer.write(context, trackingGeometry)
+
+        if outputJson:
+            writer = JsonSurfacesWriter(
+                level=acts.logging.INFO,
+                trackingGeometry=trackingGeometry,
+                outputDir=os.path.join(outputDir, "json"),
+                writePerEvent=True,
+                writeSensitive=True,
+            )
+            writer.write(context)
+
+            jmConverterCfg = MaterialMapJsonConverter.Config(
+                processSensitives=True,
+                processApproaches=True,
+                processRepresenting=True,
+                processBoundaries=True,
+                processVolumes=True,
+                context=context.geoContext,
+            )
+
+            jmw = JsonMaterialWriter(
+                level=acts.logging.VERBOSE,
+                converterCfg=jmConverterCfg,
+                fileName=os.path.join(outputDir, "json", "material"),
```

Comment:
That is really personal but I don't think putting the material in it's own json folder in needed (compare the obj/csv it is not like we use 100's of files)
Personnaly I would change this to : 
```suggestion
                fileName=os.path.join(outputDir, "geometry-map"),
``` 
Which is what we had before.